scripts/install-prometheus-operator.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+# apply the crds and wait for them
+git clone https://github.com/prometheus-operator/kube-prometheus.git
```

Comment:
Change this to: 

```suggestion
git clone https://github.com/prometheus-operator/kube-prometheus.git --depth 1 -b v0.11.0
```

This will checkout the latest release v0.11.0 from [here](https://github.com/prometheus-operator/kube-prometheus/releases/tag/v0.11.0).